protected/controllers/AdminController.php

```diff
@@ -1609,15 +1609,15 @@ public function actionEditCommissioningBodyService()
         if ($commissioning_bt) {
             $criteria->addColumnCondition(array('commissioning_body_type_id' => $commissioning_bt->id));
         }
-        $data = CHtml::listData(CommissioningBody::model()->findAll($criteria), 'id', 'name');
-        if (empty($data)) {
-          array_push($errors, array('No Local Authorities type commissioning body is defined, please contact admin user to fix this problem.'));
+        $db_list = CHtml::listData(CommissioningBody::model()->findAll($criteria), 'id', 'name');
+        if (empty($db_list)) {
+          array_push($errors, array('No commissioning body exists.'));
```

Comment:
I believe this list can be empty if there are commissioning bodies but none of them have the Local Authorities type. The original error message is probably sufficient here.

Reply:
For LA Admin page, yes it is, it only check the Local Authorities type. But this function is also called in admin page core>commissioning body services. I'm afraid it will misleading users if we use the original message.

Reply:
Good point. I suppose that makes sense.